onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2003,6 +2009,10 @@ def aten_scaled_dot_product_attention_bool_mask(
         is_causal and attn_mask is None
     ), "is_causal and attn_mask cannot be set at the same time"
 
+    assert (
+        not enable_gqa
+    ), "conversion of scaled_dot_product_attention not implemented if enable_gqa is True"
```

Comment:
Is it possible to implement? having partial implementation w/ assert is not recommended in torchlib

Reply:
I think GQA is now only accessible with contrib op though: https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#com.microsoft.GroupQueryAttention

Reply:
Should be implementable as a function?

Reply:
I guess likely the ONNX framework (but not ORT) users should have their own custom ops if they want to run a model with this op.

Reply:
https://github.com/microsoft/onnxscript/issues/1802